src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -309,21 +384,38 @@
             toggleModal() {
                 this.open = !this.open;
             },
+            confirmImport() {
+                if (this.newDatasetId && this.editsRequireConfirmation) {
```

Comment:
Is it possible to click "Import" and trigger this method if `newDataId` isn't set? (I think @EmmaLRussell may already have raised this)

Reply:
It shouldn't be - though I've just noticed we have a similar clause in importDataset so I guess it's consistent with existing code. We probably shouldn't have a policy of not trusting our own logic!

Reply:
I think you are referring to a slightly different situation previously involving `selectedDataset`. Technically, the user shouldn't be able to click the button if `newDatasetId` is not set as the button will be disable; however, I noticed that the `importDataset` method does check for `newDatasetId` so I was just following the existing pattern.